server.js

```diff
@@ -1652,6 +1652,9 @@ cache(function(data, match, sendBadge, request) {
       } else if (status === 'uptodate') {
         badgeData.colorscheme = 'brightgreen';
         status = 'up-to-date';
+      } else if (status === 'none') {
+        badgeData.colorscheme = 'brightgreen';
+        status = 'none';
```

Comment:
Thanks! :) Yeah, I guess I wasn't really looking at what I was doing, sorry for missing that. Fixed.